packages/api/cms-api/src/auth/strategies/auth-proxy-jwt.strategy.ts

```diff
@@ -30,11 +31,21 @@ export function createAuthProxyJwtStrategy({
                 }),
                 audience,
                 ...strategyOptions,
+                passReqToCallback: true,
             });
         }
 
-        async validate(data: JwtPayload): Promise<CurrentUser> {
-            return this.service.createCurrentUser(await this.service.createUserFromIdToken(data));
+        async validate(request: Request): Promise<CurrentUser> {
+            let user;
+            if (strategyOptions?.passReqToCallback) {
+                user = await this.service.createUserFromIdToken(request);
```

Comment:
Are you confident that this works? I don't know how to test this locally. But based on the types it doesn't look to me as if the request itself is a valid JWTPayload which should make this code fail:

https://github.com/vivid-planet/comet/blob/a27b55345a15684afad3461992320367d1d786d0/packages/api/cms-api/src/user-permissions/user-permissions.service.ts#L65-L67